docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

```diff
@@ -120,11 +120,13 @@ offered by Grafana.
 3. Configure the recording rule:
    1. Select `Mimir or Loki recording rule` in the top selector.
```

Comment:
```suggestion
   1. Select **Mimir or Loki recording rule** in the top selector.
```

Reply:
I was just doing copy-pasta, will apply your fixes

Reply:
Yeah, I completely understand :) thanks for considering the changes!

Apologies for the inconsistent style.

As an interim owner of the docs, I'm looking to clean bits up as I go and I appreciate any contributions that can help me do that.